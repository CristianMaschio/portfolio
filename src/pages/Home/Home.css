.home {
  margin-left: 18rem;
}

.intro {
  background-image: url("intro.png");
  background-position: left;
  background-size: cover;
  height: 22rem;
  flex: 1;
  display: flex;
  flex-direction: column;
  justify-content: center;
}

.intro h1 {
  color: white;
  text-align: center;
  font-weight: 500;
  font-size: 3rem;
  text-shadow: 2px 4px 5px black;
  padding: 0;
  margin: 0;
}

.intro h2 {
  color: white;
  text-align: center;
  font-weight: 500;
  font-size: 2rem;
  text-shadow: 2px 4px 5px black;
  padding: 0;
  margin: 0;
}

.paragraphs {
}
