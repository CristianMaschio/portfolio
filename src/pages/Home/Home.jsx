import React, { PureComponent } from "react";
import Parallax from "react-rellax";
import Paragraph from "../../components/Paragraph";
import { Fade } from "react-reveal";
import TagsCanvas from "react-tags-canvas";
import background from "../../utils/background";
import json from "../../assets/languages/En-paragraphs.json";

class Home extends PureComponent {
  componentDidMount() {
    background();
  }

  render() {
    return (
      <div className="home">
        <div className="image-background">
          <canvas id="canvas"></canvas>
        </div>
        <Parallax speed={-3}>
          <div className="intro intro-flex-center">
            <div className="intro-flex-center">
              <Fade bottom>
                <h1>Cristian Maschio</h1>
                <h2>{"<SeniorFrontEndEngineer />"}</h2>
              </Fade>
            </div>
          </div>
        </Parallax>
        <div className="paragraphs">
          <div className="waves-top">
            <svg
              fill="white"
              xmlns="http://www.w3.org/2000/svg"
              viewBox="0 0 1000 100"
              preserveAspectRatio="none"
            >
              <path d="M 0 14 s 88.64 3.48 300 36 c 260 40 514 27 703 -10 l 12 28 l 3 36 h -1018 z"></path>
              <path d="M 0 45 s 271 45.13 500 32 c 157 -9 330 -47 515 -63 v 86 h -1015 z"></path>
              <path d="M 0 58 s 188.29 32 508 32 c 290 0 494 -35 494 -35 v 45 h -1002 z"></path>
            </svg>
          </div>
          <div className="paragraph">
            <Fade bottom>
              <h2 className="paragraphTitle">Quick overview</h2>
            </Fade>
            <div className="paragraph-row">
              <ul>
                <Fade bottom cascade>
                  <li>
                    <b>5+ years of working experience</b> building web-apps for
                    clients like TransportForLondon, Google, Spotify, Roche,
                    Nike in London.
                  </li>
                  <li>
                    <b>University degree in Computer Science.</b>
                  </li>
                  <li>
                    <b>Worked in a wide range of projects</b> from existing big,
                    complex projects to build scalable projects from scratch
                    used by millions of users.
                  </li>
                  <li>
                    Worked and collaborated directly with stakeholders,
                    designers, team and multiple teams to deliver projects and
                    new features.
                  </li>
                  <li>Clean, reusable and tested code.</li>
                  <li>Attention to details, user experience.</li>
                </Fade>
              </ul>
              <Fade bottom>
                <div className="tags-canvas">
                  <TagsCanvas
                    textColour="rgba(59, 98, 196, 0.9)"
                    maxSpeed={0.05}
                    weightSize={0.45}
                    shuffleTags
                    shape="sphere"
                    noSelect
                    wheelZoom={false}
                    initial={[0.05, -0.05]}
                    tags={[
                      { value: "React", weight: 50 },
                      { value: "Javascript", weight: 30 },
                      { value: "HTML5", weight: 20 },
                      { value: "CSS3", weight: 20 },
                      { value: "Typescript", weight: 35 },
                      { value: "Git", weight: 20 },
                      { value: "Redux", weight: 30 },
                      { value: "Redux-Saga", weight: 20 },
                      { value: "NodeJS", weight: 20 },
                      { value: "NextJS", weight: 20 },
                      { value: "Docker", weight: 15 },
                      { value: "React-Native", weight: 20 },
                      { value: "Firebase", weight: 20 },
                      { value: "AWS", weight: 20 },
                      { value: "Gatsby", weight: 20 },
                      { value: "GraphQL", weight: 20 },
                    ]}
                  />
                </div>
              </Fade>
            </div>
          </div>

          {json.paragraphs?.map((paragraph, index) => {
            return (
              <Paragraph
                id={paragraph.id}
                key={index}
                index={index}
                title={paragraph.title}
                image={paragraph.image}
                sections={paragraph.sections}
                moreSections={paragraph.more}
              />
            );
          })}
        </div>
      </div>
    );
  }
}

export default Home;
