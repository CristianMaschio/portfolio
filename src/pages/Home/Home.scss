@import "../../styles/colors";

$text-shadow: 2px 4px 5px black;

.home {
  margin-left: 18rem;
}

.intro {
  background-image: url("intro.jpg");
  background-position: left;
  background-size: cover;
  height: 22rem;
  flex: 1;
  display: flex;
  flex-direction: column;
  justify-content: center;
}

.intro h1 {
  color: white;
  text-align: center;
  font-weight: 500;
  font-size: 3rem;
  text-shadow: $text-shadow;
  padding: 0;
  margin: 0;
}

.intro h2 {
  color: white;
  text-align: center;
  font-weight: 500;
  font-size: 2rem;
  text-shadow: $text-shadow;
  padding: 0;
  margin: 0;
}

.paragraphs {
  display: flex;
  flex-direction: column;
}
