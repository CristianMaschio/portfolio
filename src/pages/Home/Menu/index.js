import React, { useContext } from "react"
import Context from "../../../Context"

import whiteInfo from "../../../assets/images/whiteInfo.png"
const menuLink = require(`../../../assets/languages/En-content.json`).menu
  
export default function Menu() {
  const { setRenderInformation } = useContext(Context)
  
  return (
    <div className="menu">
      <img
        onClick={setRenderInformation}
        src={whiteInfo}
        alt="info click icon"
        className="info-menu"
      />
      <p onClick={setRenderInformation}>Cristian Maschio</p>
      <ul>
        <li>
          <a href="#professional">{menuLink.professional} </a>
        </li>
        <li>
          <a href="#education">{menuLink.education}</a>
        </li>
        <li>
          <a href="#passions">{menuLink.passions}</a>
        </li>
        <li>
          <a href="#personal-experiences">{menuLink.experiences}</a>
        </li>
        <li>
          <a href="#code">{menuLink.code}</a>
        </li>
      </ul>
    </div>
  )
}
