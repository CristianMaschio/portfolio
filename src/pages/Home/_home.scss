$text-shadow: 2px 2px 5px rgba(0, 0, 0, 0.7);

.image-background {
  position: fixed;
  top: 0;
  left: 0;
  z-index: -1;
  height: 100vh;
  width: 100vw;

  canvas {
    height: 100vh;
    width: 100vw;
  }
}

.home {
  .intro {
    min-height: 100vh;
    flex: 1;
    display: flex;
    flex-direction: column;
    justify-content: center;
    margin-top: -50px;
    .intro-flex-center {
      height: 100%;
      display: flex;
      flex-direction: column;
      justify-content: center;
      align-items: center;

      button {
        font-weight: bold;
        margin-top: 2rem;
      }

      .icons {
        display: flex;
        justify-content: space-between;
        align-items: center;
        width: 15rem;
        margin-top: 1.8rem;
        align-self: center;
        a {
          color: white;
          font-size: 1.8rem;
        }
      }
    }
    h1 {
      color: white;
      text-align: center;
      font-weight: 500;
      font-size: 3rem;
      text-shadow: $text-shadow;
      padding: 0;
      margin: 0;

      @media screen and (max-width: $mobile-max-width) {
        font-size: 2rem;
      }
    }
    h2 {
      color: white;
      text-align: center;
      font-weight: 500;
      font-size: 2rem;
      text-shadow: $text-shadow;
      padding: 0;
      margin: 0;
      margin-top: 0.2rem;

      @media screen and (max-width: $mobile-max-width) {
        font-size: 1.5rem;
      }
    }
  }
  .paragraphs {
    position: relative;
    display: flex;
    flex-direction: column;
    align-items: center;
    margin-top: 0;
    background-color: $background-secondary-color;
  }
  .p-des {
    margin: 0;
    text-align: center;
    max-width: 900px;
    padding: 2rem 2rem 0 2rem;
  }
}

.waves {
  height: 100px;
  width: 100%;
  z-index: 2;
  pointer-events: none;

  svg {
    height: 100px;
    width: 100%;
  }
  svg path:first-child {
    opacity: 0.15;
  }

  svg path:nth-child(2) {
    opacity: 0.3;
  }

  @media screen and (max-width: $mobile-max-width) {
    height: 50px;

    svg {
      height: 50px;
    }
  }
}

.waves-top {
  position: absolute;
  top: -99px;
  left: 0;

  @media screen and (max-width: $mobile-max-width) {
    top: -49px;
  }
}
