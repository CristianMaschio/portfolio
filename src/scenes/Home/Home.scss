@import "../../data/styles/colors";

$text-shadow: 2px 4px 5px black;

.home {
  margin-left: 18rem;

  @media screen and (max-width: $mobile-max-width) {
    margin-left: 0;
  }

  .intro {
    background-image: url("../../data/images/intro.jpg");
    background-position: left;
    background-size: cover;
    height: 22rem;
    flex: 1;
    display: flex;
    flex-direction: column;
    justify-content: center;
    .intro-white-trasparence {
      height: 100%;
      display: flex;
      flex-direction: column;
      justify-content: center;
      background: rgba(255, 255, 255, 0.1);
    }
    h1 {
      color: white;
      text-align: center;
      font-weight: 500;
      font-size: 3rem;
      text-shadow: $text-shadow;
      padding: 0;
      margin: 0;
    }
    h2 {
      color: white;
      text-align: center;
      font-weight: 500;
      font-size: 2rem;
      text-shadow: $text-shadow;
      padding: 0;
      margin: 0;
    }
  }
  .paragraphs {
    display: flex;
    flex-direction: column;
    margin-top: -3rem;
  }
}

.foolishIn {
  animation-duration: 3s;
  -webkit-animation-name: foolishIn;
  animation-name: foolishIn;
}
@-webkit-keyframes foolishIn {
  0% {
    opacity: 0;
    -webkit-transform-origin: 50% 50%;
    transform-origin: 50% 50%;
    -webkit-transform: scale(0, 0) rotate(360deg);
    transform: scale(0, 0) rotate(360deg);
  }

  20% {
    opacity: 1;
    -webkit-transform-origin: 0% 100%;
    transform-origin: 0% 100%;
    -webkit-transform: scale(0.5, 0.5) rotate(0deg);
    transform: scale(0.5, 0.5) rotate(0deg);
  }

  40% {
    opacity: 1;
    -webkit-transform-origin: 100% 100%;
    transform-origin: 100% 100%;
    -webkit-transform: scale(0.5, 0.5) rotate(0deg);
    transform: scale(0.5, 0.5) rotate(0deg);
  }

  60% {
    opacity: 1;
    -webkit-transform-origin: 0%;
    transform-origin: 0%;
    -webkit-transform: scale(0.5, 0.5) rotate(0deg);
    transform: scale(0.5, 0.5) rotate(0deg);
  }

  80% {
    opacity: 1;
    -webkit-transform-origin: 0% 0%;
    transform-origin: 0% 0%;
    -webkit-transform: scale(0.5, 0.5) rotate(0deg);
    transform: scale(0.5, 0.5) rotate(0deg);
  }

  100% {
    opacity: 1;
    -webkit-transform-origin: 50% 50%;
    transform-origin: 50% 50%;
    -webkit-transform: scale(1, 1) rotate(0deg);
    transform: scale(1, 1) rotate(0deg);
  }
}
@keyframes foolishIn {
  0% {
    opacity: 0;
    -webkit-transform-origin: 50% 50%;
    transform-origin: 50% 50%;
    -webkit-transform: scale(0, 0) rotate(360deg);
    transform: scale(0, 0) rotate(360deg);
  }

  20% {
    opacity: 1;
    -webkit-transform-origin: 0% 100%;
    transform-origin: 0% 100%;
    -webkit-transform: scale(0.5, 0.5) rotate(0deg);
    transform: scale(0.5, 0.5) rotate(0deg);
  }

  40% {
    opacity: 1;
    -webkit-transform-origin: 100% 100%;
    transform-origin: 100% 100%;
    -webkit-transform: scale(0.5, 0.5) rotate(0deg);
    transform: scale(0.5, 0.5) rotate(0deg);
  }

  60% {
    opacity: 1;
    -webkit-transform-origin: 0%;
    transform-origin: 0%;
    -webkit-transform: scale(0.5, 0.5) rotate(0deg);
    transform: scale(0.5, 0.5) rotate(0deg);
  }

  80% {
    opacity: 1;
    -webkit-transform-origin: 0% 0%;
    transform-origin: 0% 0%;
    -webkit-transform: scale(0.5, 0.5) rotate(0deg);
    transform: scale(0.5, 0.5) rotate(0deg);
  }

  100% {
    opacity: 1;
    -webkit-transform-origin: 50% 50%;
    transform-origin: 50% 50%;
    -webkit-transform: scale(1, 1) rotate(0deg);
    transform: scale(1, 1) rotate(0deg);
  }
}
