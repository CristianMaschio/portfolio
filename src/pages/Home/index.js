import React, { Component } from "react"
import "./Home.css"

import Paragraph from "../_shared/Paragraph"
import data from "../../data/data"

class Home extends Component {
  render() {
    return (
      <div className="home">
        <div className="intro">
          <h1>Cristian Maschio</h1>
          <h2>Software Developer</h2>
        </div>
        <div className="paragraphs">
          {data.paragraphs.map((paragraph, index) => {
            return (
              <Paragraph
                key={index}
                index={index}
                title={paragraph.title}
                image={paragraph.image}
                sections={paragraph.sections}
              />
            )
          })}
        </div>
      </div>
    )
  }
}

export default Home
