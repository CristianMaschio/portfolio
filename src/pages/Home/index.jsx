import React, { PureComponent } from "react";
import Paragraph from "./Paragraph";
import Menu from "./Menu";
import Rellax from "rellax";

class Home extends PureComponent {
  state = {
    paragraphs: undefined,
  };

  componentDidMount() {
    this.setState({
      paragraphs: require(`../../assets/languages/En-paragraphs.json`)
        .paragraphs,
    });
    new Rellax(".animate", {
      speed: -3,
    });
  }

  render() {
    return (
      <div className="home">
        <div className="intro intro-white-trasparence">
          <div className="intro-white-trasparence">
            <div className="foolishIn animate">
              <h1>Cristian Maschio</h1>
              <h2>{"<SoftwareEngineer />"}</h2>
            </div>
          </div>
        </div>
        <Menu paragraphs={this.state.paragraphs} />
        <div className="paragraphs">
          {this.state.paragraphs?.map((paragraph, index) => {
            return (
              <Paragraph
                id={paragraph.id}
                key={index}
                index={index}
                title={paragraph.title}
                image={paragraph.image}
                sections={paragraph.sections}
                moreSections={paragraph.more}
              />
            );
          })}
        </div>
      </div>
    );
  }
}

export default Home;
