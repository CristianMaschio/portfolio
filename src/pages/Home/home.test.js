import React from 'react'
import rederer from 'react-test-renderer'
import Home from '.'

it('Home: render correctly', ()=> {
  const home = rederer.create(<Home />).toJSON()

  expect(home).toMatchSnapshot()
})