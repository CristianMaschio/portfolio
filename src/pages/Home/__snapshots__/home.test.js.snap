// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home: render correctly 1`] = `
<div
  className="home"
>
  <div
    className="intro intro-white-trasparence"
  >
    <div
      className="intro-white-trasparence"
    >
      <div
        className="foolishIn"
      >
        <h1>
          Cristian Maschio
        </h1>
        <h2>
          &lt;SoftwareEngineer /&gt;
        </h2>
      </div>
    </div>
  </div>
  <div
    className="menu"
  >
    <span
      aria-label="info-circle"
      className="anticon anticon-info-circle info-menu"
      onClick={[Function]}
      role="img"
      tabIndex={-1}
    >
      <svg
        aria-hidden="true"
        className=""
        data-icon="info-circle"
        fill="currentColor"
        focusable="false"
        height="1em"
        viewBox="64 64 896 896"
        width="1em"
      >
        <path
          d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm32 664c0 4.4-3.6 8-8 8h-48c-4.4 0-8-3.6-8-8V456c0-4.4 3.6-8 8-8h48c4.4 0 8 3.6 8 8v272zm-32-344a48.01 48.01 0 010-96 48.01 48.01 0 010 96z"
        />
      </svg>
    </span>
    <p
      onClick={[Function]}
    >
      Cristian Maschio
    </p>
    <ul>
      <li>
        <a
          href="#professional"
        >
          Professional
        </a>
      </li>
      <li>
        <a
          href="#education"
        >
          Education
        </a>
      </li>
      <li>
        <a
          href="#passion"
        >
          Passion
        </a>
      </li>
      <li>
        <a
          href="#experience"
        >
          Experience
        </a>
      </li>
      <li>
        <a
          href="#code"
        >
          Code
        </a>
      </li>
    </ul>
  </div>
  <div
    className="paragraphs"
  >
    <div
      id="professional"
    >
      <div
        className="paragraph"
      >
        <div
          className="rightLine"
        />
        <div
          className="paragraphContainer"
        >
          <img
            alt="Universtity of Padua logo"
            className="paragraphImage"
            src="work.png"
          />
          <h2
            className="paragraphTitle"
          >
            Professional
          </h2>
          <img
            alt="Universtity of Padua logo"
            className="paragraphImage"
            src="work.png"
          />
        </div>
        <div
          className="paragraphContainer"
        >
          <div
            className="sectionsContainer"
          >
            <div
              className="sectionContainer"
            >
              <div
                className="sectionContent description"
              >
                <div
                  className="section-image"
                >
                  <img
                    alt="Rehab - Full Stack Developer"
                    src={
                      Object {
                        "ReactComponent": Object {
                          "$$typeof": Symbol(react.forward_ref),
                          "render": [Function],
                        },
                        "__esModule": true,
                        "default": "rehab.svg",
                      }
                    }
                  />
                </div>
                <p
                  className="date"
                >
                  12/2019 - Current
                </p>
                <h3>
                  Rehab - Full Stack Developer
                </h3>
                <div
                  className="multy-section"
                >
                  <div
                    className="sub-section"
                  >
                    <div
                      className="sub-section-title"
                    >
                      Technologies:
                    </div>
                    <div
                      className="sub-section-content"
                      dangerouslySetInnerHTML={
                        Object {
                          "__html": "React, Javascript, Redux, Html, Scss, Python & Django.",
                        }
                      }
                    />
                  </div>
                  <div
                    className="sub-section"
                  >
                    <div
                      className="sub-section-title"
                    >
                      Achieving:
                    </div>
                    <div
                      className="sub-section-content"
                      dangerouslySetInnerHTML={
                        Object {
                          "__html": "Working in projects with short deadlines, integrating new features in large platforms and learned to better manage Git flow",
                        }
                      }
                    />
                  </div>
                </div>
                <div
                  className="link-container"
                >
                  <a
                    href="https://www.rehabagency.ai/"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    Go to page &gt;&gt;
                  </a>
                </div>
              </div>
            </div>
            <div
              className="sectionContainer"
            >
              <div
                className="sectionContent description"
              >
                <div
                  className="section-image"
                >
                  <img
                    alt="GestionaleAuto.com - React Developer"
                    src="gestionaleauto.png"
                  />
                </div>
                <p
                  className="date"
                >
                  05/2019 - 30/09/2019
                </p>
                <h3>
                  GestionaleAuto.com - React Developer
                </h3>
                <div
                  className="multy-section"
                >
                  <div
                    className="sub-section"
                  >
                    <div
                      className="sub-section-title"
                    >
                      Technologies:
                    </div>
                    <div
                      className="sub-section-content"
                      dangerouslySetInnerHTML={
                        Object {
                          "__html": "React, Javascript, Redux, Redux-Form, Redux-Saga, Html & Scss.",
                        }
                      }
                    />
                  </div>
                  <div
                    className="sub-section"
                  >
                    <div
                      className="sub-section-title"
                    >
                      Achieving:
                    </div>
                    <div
                      className="sub-section-content"
                      dangerouslySetInnerHTML={
                        Object {
                          "__html": "Learned to working in Agile Scrum team: my daily tasks ranged from refactoring to develop new feature for a large project code base of a management software. In which I managed to improve the simplicity and performance of some components, making them 30-40% faster.",
                        }
                      }
                    />
                  </div>
                </div>
                <div
                  className="link-container"
                >
                  <a
                    href="https://gestionaleauto.com/"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    Go to page &gt;&gt;
                  </a>
                </div>
              </div>
            </div>
            <div
              className="sectionContainer"
            >
              <div
                className="sectionContent description"
              >
                <div
                  className="section-image"
                >
                  <img
                    alt="Discontinuity - Full Stack Developer"
                    src="discontinuity.png"
                  />
                </div>
                <p
                  className="date"
                >
                  10/2018 - 04/2019
                </p>
                <h3>
                  Discontinuity - Full Stack Developer
                </h3>
                <div
                  className="multy-section"
                >
                  <div
                    className="sub-section"
                  >
                    <div
                      className="sub-section-title"
                    >
                      Technologies:
                    </div>
                    <div
                      className="sub-section-content"
                      dangerouslySetInnerHTML={
                        Object {
                          "__html": "React Native, React, Javascript, Gatsby, AWS, DynamoDB, GraphQL, Html & Scss.",
                        }
                      }
                    />
                  </div>
                  <div
                    className="sub-section"
                  >
                    <div
                      className="sub-section-title"
                    >
                      Achieving:
                    </div>
                    <div
                      className="sub-section-content"
                      dangerouslySetInnerHTML={
                        Object {
                          "__html": "Working in Agile team, with maintaining and implementing applications and web sites from scratch",
                        }
                      }
                    />
                  </div>
                </div>
                <div
                  className="link-container"
                >
                  <a
                    href="http://discontinuity.eu/"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    Go to page &gt;&gt;
                  </a>
                </div>
              </div>
            </div>
            <div
              className="more-container"
            />
          </div>
        </div>
      </div>
      
    </div>
    <div
      id="education"
    >
      <div
        className="paragraph"
      >
        <div
          className="rightLine"
        />
        <div
          className="paragraphContainer"
        >
          <img
            alt="Universtity of Padua logo"
            className="paragraphImage"
            src="education.png"
          />
          <h2
            className="paragraphTitle"
          >
            Education
          </h2>
          <img
            alt="Universtity of Padua logo"
            className="paragraphImage"
            src="education.png"
          />
        </div>
        <div
          className="paragraphContainer"
        >
          <div
            className="sectionsContainer"
          >
            <div
              className="sectionContainer"
            >
              <div
                className="sectionContent description"
              >
                <p
                  className="date"
                >
                  2015 - 2019
                </p>
                <h3>
                  University of Padua - Computer Science
                </h3>
                <div
                  className="multy-section"
                >
                  <div
                    className="sub-section"
                  >
                    <div
                      className="sub-section-title"
                    >
                      Technologies:
                    </div>
                    <div
                      className="sub-section-content"
                      dangerouslySetInnerHTML={
                        Object {
                          "__html": "C++, SQL, Java, Javascript, Php, html, css, React Native & React.",
                        }
                      }
                    />
                  </div>
                  <div
                    className="sub-section"
                  >
                    <div
                      className="sub-section-title"
                    >
                      Achieving:
                    </div>
                    <div
                      className="sub-section-content"
                      dangerouslySetInnerHTML={
                        Object {
                          "__html": "It gave me the mental flexibility to learn new programming languages quickly, handle pointers, object-oriented and concurrent programming.",
                        }
                      }
                    />
                  </div>
                </div>
              </div>
            </div>
            <div
              className="more-container"
            />
          </div>
        </div>
      </div>
      
    </div>
    <div
      id="passion"
    >
      <div
        className="paragraph"
      >
        <div
          className="rightLine"
        />
        <div
          className="paragraphContainer"
        >
          <img
            alt="Universtity of Padua logo"
            className="paragraphImage"
            src="passions.png"
          />
          <h2
            className="paragraphTitle"
          >
            Passion
          </h2>
          <img
            alt="Universtity of Padua logo"
            className="paragraphImage"
            src="passions.png"
          />
        </div>
        <div
          className="paragraphContainer"
        >
          <div
            className="sectionsContainer"
          >
            <div
              className="sectionContainer"
            >
              <div
                className="sectionContent description"
              >
                <p
                  className="date"
                >
                  -
                </p>
                <h3>
                  Programming
                </h3>
                <div
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "I love programming, thinking and using my mind to create more and more clean and especially performed code. I like working with innovative technologies, mainly with web and mobile application. Currently I use a lot React and React-Native.",
                    }
                  }
                />
              </div>
            </div>
            <div
              className="sectionContainer"
            >
              <div
                className="sectionContent description"
              >
                <p
                  className="date"
                >
                  -
                </p>
                <h3>
                  Vegetable garden
                </h3>
                <div
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "These passion allow me to get my hands dirty, free my mind and appreciate how much nature can give us.",
                    }
                  }
                />
              </div>
            </div>
            <div
              className="more-container"
            />
          </div>
        </div>
      </div>
      
    </div>
    <div
      id="experience"
    >
      <div
        className="paragraph"
      >
        <div
          className="rightLine"
        />
        <div
          className="paragraphContainer"
        >
          <img
            alt="Universtity of Padua logo"
            className="paragraphImage"
            src="experiences.png"
          />
          <h2
            className="paragraphTitle"
          >
            Experience
          </h2>
          <img
            alt="Universtity of Padua logo"
            className="paragraphImage"
            src="experiences.png"
          />
        </div>
        <div
          className="paragraphContainer"
        >
          <div
            className="sectionsContainer"
          >
            <div
              className="sectionContainer"
            >
              <div
                className="sectionContent description"
              >
                <p
                  className="date"
                >
                  11/2019 - Current
                </p>
                <h3>
                  Volunteer help organizer - (GDG) Cloud London
                </h3>
                <div
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "<b>Google Developer Group</b> Cloud London is the most active developer group on the meetup platform in the world! </br> We organise hacks several times a week every week. We have a general talk once a month. And we usually have several workshops once a month. </br>  We strive to learn and share knowledge about challenging topics in a casual friendly respectful atmosphere.",
                    }
                  }
                />
                <div
                  className="link-container"
                >
                  <a
                    href="https://www.meetup.com/it-IT/gdgcloud/"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    Go to page &gt;&gt;
                  </a>
                </div>
              </div>
            </div>
            <div
              className="sectionContainer"
            >
              <div
                className="sectionContent description"
              >
                <p
                  className="date"
                >
                  21/03/2019 & 29/11/2018
                </p>
                <h3>
                  {Codemotion} Rome & {Codemotion} Milan
                </h3>
                <div
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "(Participated) Codemotion is the biggest tech conference in Italy and one of the most important in Europe. Here I had the chance to meet new people who shared my passions and follow front-end and react talks which allowed me to improve the quality of my code and learn new things.",
                    }
                  }
                />
                <div
                  className="link-container"
                >
                  <a
                    href="https://about.codemotion.com/"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    Go to page &gt;&gt;
                  </a>
                </div>
              </div>
            </div>
            <div
              className="sectionContainer"
            >
              <div
                className="sectionContent description"
              >
                <p
                  className="date"
                >
                  26/05/2018
                </p>
                <h3>
                  Hackathon won (React + Typescript)
                </h3>
                <div
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "First prize winning with a web application developed with the use of the <b>React</b> library and the <b>Google Maps API.</b>",
                    }
                  }
                />
                <div
                  className="link-container"
                >
                  <a
                    href="https://github.com/CristianMaschio/FE-ProgettoCastelfrancoVeneto"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    Go to page &gt;&gt;
                  </a>
                </div>
              </div>
            </div>
            <div
              className="sectionContainer"
            >
              <div
                className="sectionContent description"
              >
                <p
                  className="date"
                >
                  10/2017 - 07/2018
                </p>
                <h3>
                  University project - Social web app (Typescript + React + NodeJs + Jest + CircleCi)
                </h3>
                <div
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "This project gave me the opportunity to collaborate with 6 other unknown people, including a freelancer who gave me a solid foundation of React.",
                    }
                  }
                />
              </div>
            </div>
            <div
              className="sectionContainer"
            >
              <div
                className="sectionContent description"
              >
                <p
                  className="date"
                >
                  2018
                </p>
                <h3>
                  University project - Mobile app (React Native)
                </h3>
                <div
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "Here on my voluntary request for the Mobile Programming and Multimedia course, I decided to create an application with React Native, to discover the mobile world with React and learning the basics of React Native.",
                    }
                  }
                />
                <div
                  className="link-container"
                >
                  <a
                    href="https://github.com/CristianMaschio/Progetto_MPM_React_Native"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    Go to page &gt;&gt;
                  </a>
                </div>
              </div>
            </div>
            <div
              className="sectionContainer"
            >
              <div
                className="sectionContent description"
              >
                <p
                  className="date"
                >
                  24/11/2018
                </p>
                <h3>
                  DevFest Veneto - GDG Venice
                </h3>
                <div
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "I partecipated to the DevFest organized by Google Developers Group Venice.",
                    }
                  }
                />
                <div
                  className="link-container"
                >
                  <a
                    href="https://gdg-venezia.github.io/devfest-veneto-18/"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    Go to page &gt;&gt;
                  </a>
                </div>
              </div>
            </div>
            <div
              className="more-container"
            >
              <div
                className="sectionContainer"
              >
                <div
                  className="sectionContent description"
                >
                  <p
                    className="date"
                  >
                    2018 - Current
                  </p>
                  <h3>
                    Coding Gym - Programmers in Padua
                  </h3>
                  <div
                    dangerouslySetInnerHTML={
                      Object {
                        "__html": "I partecipated to some \\"4-5\\" Coding Gym. Coding Gym is an international format of programming laboratories open to any language and developed for continuous improvement.",
                      }
                    }
                  />
                  <div
                    className="link-container"
                  >
                    <a
                      href="https://www.programmersinpadua.it/"
                      rel="noopener noreferrer"
                      target="_blank"
                    >
                      Go to page &gt;&gt;
                    </a>
                  </div>
                </div>
              </div>
              <div
                className="sectionContainer"
              >
                <div
                  className="sectionContent description"
                >
                  <p
                    className="date"
                  >
                    2018 - Current
                  </p>
                  <h3>
                    Talks - Programmers in Padua
                  </h3>
                  <div
                    dangerouslySetInnerHTML={
                      Object {
                        "__html": "I partecipated to some talks. Programmers in Padua is a little community, who work and passionate about programming, new technologies and startups. We meet once a month in an open and informal environment, where the participants propose themselves for the talks.",
                      }
                    }
                  />
                  <div
                    className="link-container"
                  >
                    <a
                      href="https://www.programmersinpadua.it/"
                      rel="noopener noreferrer"
                      target="_blank"
                    >
                      Go to page &gt;&gt;
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <p
        className="more-paragraph"
        onClick={[Function]}
      >
        more
      </p>
    </div>
    <div
      id="code"
    >
      <div
        className="paragraph"
      >
        <div
          className="rightLine"
        />
        <div
          className="paragraphContainer"
        >
          <img
            alt="Universtity of Padua logo"
            className="paragraphImage"
            src="work.png"
          />
          <h2
            className="paragraphTitle"
          >
            Code
          </h2>
          <img
            alt="Universtity of Padua logo"
            className="paragraphImage"
            src="work.png"
          />
        </div>
        <div
          className="paragraphContainer"
        >
          <div
            className="sectionsContainer"
          >
            <div
              className="sectionContainer"
            >
              <div
                className="sectionContent description"
              >
                <p
                  className="date"
                >
                  10/2019
                </p>
                <h3>
                  Dynamic form - (React + Jest)
                </h3>
                <div
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "Using React, create a simple one page application with a dynamic form that takes its structure from the JSON...",
                    }
                  }
                />
                <div
                  className="link-container"
                >
                  <a
                    href="https://codesandbox.io/embed/proud-sun-bs4lc?fontsize=14"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    Go to page &gt;&gt;
                  </a>
                </div>
              </div>
            </div>
            <div
              className="sectionContainer"
            >
              <div
                className="sectionContent description"
              >
                <p
                  className="date"
                >
                  10/2019
                </p>
                <h3>
                  Manage large data - (React)
                </h3>
                <div
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "Using React, manage a larger sets of data using a pre-defined list of Employees. </br> For every Employee the rendered list shows 'firstname', 'lastname', 'company' and an icon with Employee's initials. </br> User can search Employees by 'firstname', 'lastname' and 'company'. ",
                    }
                  }
                />
                <div
                  className="link-container"
                >
                  <a
                    href="https://codesandbox.io/embed/funny-feynman-ejc7b?fontsize=14"
                    rel="noopener noreferrer"
                    target="_blank"
                  >
                    Go to page &gt;&gt;
                  </a>
                </div>
              </div>
            </div>
            <div
              className="more-container"
            />
          </div>
        </div>
      </div>
      
    </div>
  </div>
</div>
`;
